### ISSUE TITLE
Add more process bars

### ISSUE THREAD

**Issue by niekdejonge** [iomega/ms2query#88]
Right now process bars are shown for some steps, but not for every step. e.g. analog search calculating MS2Ds embeddings.
Make sure that the process bars show this well and discuss with Florian if Spec2Vec can be changed so it does not print a comment every time some peaks are missing in the training data (since this is a bit confusing and makes it hard to follow the progress)

**Comment by florian-huber** [comment]
Brief thought on this: If MS2Query will later be run from a backend (with a frontend app), then progress bars won't be needed. Rather we would have to return the progress to be displayed elsewhere.

To the other point: Spec2Vec logging will be coming soon (I hope).

**Comment by niekdejonge** [comment]
Great! I saw the updated spec2vec logging. See https://github.com/matchms/matchms/pull/271, I will try to implement this in MS2Query. 

And good point, by the time the frontend app is in this stage, we can decide how we can best implement this.

**Comment by niekdejonge** [comment]
The progress bars are fine for now. With #127 the spec2vec errors have been removed
